worker/src/main/java/gov/cms/ab2d/worker/processor/JobPreProcessorImpl.java

```diff
@@ -42,6 +58,16 @@ public Job preprocess(String jobUuid) {
             throw new IllegalArgumentException(errMsg);
         }
 
+        if (job.getSince() != null) {
```

Comment:
Really not fond of null reference to enums.  In general, prefer to have an enum value of NONE.  That enables a lot of nice delegation patters that the null makes too complicated.

For this specific case with existing data, I understand the pushback.